lib/lndclient/LndClient.ts

```diff
@@ -705,8 +733,11 @@ class LndClient extends SwapClient {
     let inactiveBalance = 0;
     let totalOutboundAmount = 0;
     let totalInboundAmount = 0;
+    const balancePerChannel = new Map<string, Balance>();
     channels.toObject().channelsList.forEach((channel) => {
       if (channel.active) {
+        balancePerChannel.set(channel.channelPoint, { localBalance: channel.localBalance, remoteBalance: channel.remoteBalance });
```

Comment:
nit: Looks like `channel` is being redundantly used here. We could destructure the channel object above in order to avoid it:
```
if (active) {
  balancePerChannel.set(channelPoint, { localBalance, remoteBalance });
}
```